src/rgw/rgw_lc_tier.cc

```diff
@@ -568,42 +568,42 @@ class RGWLCStreamObjToCloudMultipartPartCR : public RGWCoroutine {
   std::unique_ptr<rgw::sal::RGWBucket> dest_bucket;
   std::unique_ptr<rgw::sal::RGWObject> dest_obj;
 
+  rgw_lc_obj_properties obj_properties;
+  RGWBucketInfo b;
+  string target_obj_name;
+  int ret = 0;
```

Comment:
there's already a `RGWCoroutine::retcode` that you can use instead of `ret`. the only thing special about `retcode` is that it gets set automatically with the result of a `yield call`

Reply:
done.